### PR TITLE
FIX build script

### PR DESCRIPTION
**Pull request by Lexxus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.5.1
```
Comment (moelders) [inline CHANGELOG.md]: Do not add the version.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adidas/yarn-design-system",
-  "version": "1.5.0",
+  "version": "1.5.1",
```
Comment (moelders) [inline package.json]: Do not update the version.

File: package.json [hunk 3]
```diff
@@ -65,12 +65,12 @@
     "clean": "npm run clean:dist && npm run clean:example && npm run clean:scss",
     "clean:dist": "del dist",
     "clean:example": "del docs && del .nuxt",
-    "clean:scss": "del src/scss",
+    "clean:scss": "del-cli src/scss",
```
Comment (moelders) [inline package.json]: `del-cli` has to be added as `devDep` and also update the command on `clean:dist` and `clean:example`.

Comment (Lexxus) [reply package.json]: `del-cli` is already in `devDependency`.

**Pull request merged**